examples/shouldi/shouldi/golangci_lint.py

```diff
@@ -0,0 +1,35 @@
+import json
+import asyncio
+from typing import Dict, Any
+
+from dffml.df.base import op
+from dffml.df.types import Definition
+
+package_src_dir = Definition(name="package_src_dir", primitive="str")
+golangci_lint_output = Definition(
+    name="golangci_lint_output", primitive="Dict[str, Any]"
+)
+
+
+@op(inputs={"pkg": package_src_dir}, outputs={"report": golangci_lint_output})
+async def run_golangci_lint(pkg: str) -> Dict[str, Any]:
+    """
+    CLI usage: dffml service dev run -log debug shouldi.golangci_lint:run_golangci_lint -pkg .
+    """
+    proc = await asyncio.create_subprocess_exec(
+        "golangci-lint",
```

Comment:
`golangci-lint` is not installed because of that `examples/shouldi` gives error. To fix that
```
steps:
    - uses: actions/checkout@v1
    - name: Set up Python ${{ matrix.python-version }}
      uses: actions/setup-python@v1
      with:
        python-version: ${{ matrix.python-version }}
    - name: Install dependencies
      run: |
        mkdir -p ~/.local/bin
        python -m pip install --upgrade pip twine
        pip install -U -e .[dev]
        if [ "${{ matrix.plugin }}" == "feature/git" ]; then
          curl -sSL https://github.com/XAMPPRocky/tokei/releases/download/v9.1.1/tokei-v9.1.1-x86_64-unknown-linux-gnu.tar.gz | tar xvz -C "$HOME/.local/bin/"
          sudo apt-get update && sudo apt-get install -y git subversion cloc openssl
        fi
        if [ "${{ matrix.plugin }}" == "source/mysql" ]; then
          curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
          sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable"
          sudo apt-get update && sudo apt-get install -y docker-ce docker-ce-cli containerd.io
          docker pull mariadb:10
        fi
```
add the below codes, to end of the code above
```
        if [ "${{ matrix.plugin }}" == "examples/shouldi" ]; then
                  curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s v1.23.6
        fi
```
 in `.github/workflows/testing.yml`